elizhahn/overlook
# Feature/manager login

# Is this a fix or a feature?
- feature
# What is the change?
- a manager feature was added
- now there is a manager login, and interactive manager dashboard
- A manager can: 
   - login
   - search users by name
   - see a snapshot of the day
   - book a room for a user
  
# What does it fix?
- the style files were refactored
- The index.js file was reorganized logically
- All files were cleaned up with a linter
# Where should the reviewer start?
- index.js
# How should this be tested?
- login to the manager dashboard with "manager"
- A dashboard should appear
- you should be able to search users by name and see their history
- you should be able to see a snap shot for the day
- you should be able to search rooms by date
- you should be able to book a room for a user and the room should appear on the user's list
# Screenshots (if appropriate):
